ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml

```diff
@@ -1,5 +1,31 @@
 postsubmits:
   openshift/ci-tools:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: arm01
+    decorate: true
+    labels:
+      ci-operator.openshift.io/cluster: arm01
+    max_concurrency: 1
+    name: branch-ci-openshift-ci-tools-master-build-ci-operator-arm64
+    spec:
+      containers:
+      - args:
+        - --namespace=ci
+        - start-build
+        - --follow
+        - buildconfig/ci-operator-arm64
+        command:
+        - oc
+        image: image-registry.openshift-image-registry.svc:5000/ocp-arm64/4.13:cli
```

Comment:
How can we be sure that this job will run on an arm node?

Reply:
```yaml
cluster: arm01
...
labels:
  ci-operator.openshift.io/cluster: arm01
```
will ensure Prow is going to spawn the pod on the arm cluster.
`image-registry.openshift-image-registry.svc:5000/ocp-arm64/4.13:cli` holds `oc` version `arm64` which will authenticate to the cluster as `ci-images-builder`.

Reply:
We have both amd64 and arm64 nodes in the arm01 cluster. We need to make sure that the pod will be spawned in the arm64 node.